spec/helpers/period_helper_helper_spec.rb

```diff
@@ -9,9 +9,9 @@
     let(:current_date) { Date.current }
 
     it 'generates a sorted list of dates given a quarter and time range' do
-      expect(period_list_as_dates(:quarter, 3)).to eq([current_date.beginning_of_month,
-                                                       current_date.beginning_of_month - 3.months,
-                                                       current_date.beginning_of_month - 6.months])
+      expect(period_list_as_dates(:quarter, 3)).to eq([current_date.beginning_of_quarter,
```

Comment:
This test was breaking because it was just wrong (no tz issues per se). The code was correct, the test was wrong.